packages/react-storage/src/components/StorageBrowser/Controller.tsx

```diff
@@ -23,5 +27,12 @@ export function Controller(): null {
     }
   }, [handleListLocations, loadLocations]);
 
+  React.useEffect(() => {
+    if (isRefreshing && !loadLocations) {
```

Comment:
how can I signal to `Controller` to start refreshing again? I was thinking of using this `isRefreshing` state, but if there's a better way then please let me know